rts/Game/Camera/FPSController.cpp

```diff
@@ -64,9 +64,8 @@ void CFPSController::KeyMove(float3 move)
 void CFPSController::MouseMove(float3 move)
 {
 	RECOIL_DETAILED_TRACY_ZONE;
-	camera->SetRotY(camera->GetRot().y + mouseScale * move.x);
-	camera->SetRotX(std::clamp(camera->GetRot().x + mouseScale * move.y * move.z, 0.01f, math::PI * 0.99f));
-	dir = camera->GetDir();
+	rot.y += mouseScale * move.x;
+	rot.x = std::clamp(rot.x + mouseScale * move.y * move.z, 0.01f, math::PI * 0.99f);
```

Comment:
so what's the difference between `rot` and `dir`?

Reply:
`rot` is the rotation vector, encoding pitch, yaw and roll. `dir` is the 3d direction vector.